### ISSUE TITLE
iOS - Read Error: Code 241

### ISSUE THREAD

**Issue by tfmontague** [randdusing/cordova-plugin-bluetoothle#277]

**Comment by tfmontague** [comment]
This might be caused by the Peripheral not responding to a read request in time...

http://stackoverflow.com/questions/33299794/why-does-it-take-longer-for-cbcharacteristics-to-receive-a-write-value-than-it-d

**Comment by randdusing** [comment]
Are you connecting to a device that's using the server side of the plugin? Are you sure it's responding to the proper requestId?

**Comment by tfmontague** [comment]
I've only been able to generate this error once. So it's not as common as the #275 and #278 errors.
For all three errors seen in #275, #277, and #278. The setup is the same.

Device A - runs a client
Device B - runs a server

The client and server connect, services and characteristics on the server are discovered, and at times the client will `read` and the server will `respond`. Most the time

Haven't researched the `requestId` to understand the role it plays in `read`. But I'm using `requestId: 0`, as seen in the server code that I provided in #275 .
